test/dbus.c

```diff
@@ -296,6 +371,56 @@ TEST test_invalid_notification(void)
         PASS();
 }
 
+TEST test_dbus_cb_dunst_Properties_Set(void)
```

Comment:
I didn't add a test for `dbus_cb_dunst_Properties_Get`, just `dbus_cb_dunst_Properties_Set`.